client/src/Components/ErrorPage.jsx

```diff
@@ -21,9 +21,7 @@ function ErrorPage() {
     <>
       <Grid item />
       <Grid item xs={12}>
-        <Paper className={classes.paper}>
-          Oh no! You&apos;ve found our 404 page!
-        </Paper>
+        <Paper className={classes.paper}>You found our 404 page!</Paper>
```

Comment:
What was once framed as a bad thing is now a beautiful discovery. The joy of finding a 404!

Reply:
It throws weird errors about no &apos;